### ISSUE TITLE
SortableListener postRemove event crashes large DB (ORM adapter)

### ISSUE THREAD

**Issue by forgie1** [doctrine-extensions/DoctrineExtensions#2270]
If there is DB with:

- hundred of thousands of entities with positions e.g. 0, 1, 2, ..., 658541, -- all in one group
- and e.g. 4.000 random entities are removed in one `EM::flush()`

then DB goes down in SortableListener postRemove event. `ORM::updatePositions()` was used.
The crash was severe - tested on MariaDB 10.3 also 10.4. On Linux and Windows the same.
DB was even not recoverable after this happen, DB had to be deleted and recreated.

Basically Sortable works also without relocations persisted on postRemove, but this (disabling the postRemove event in SortableListener) is not the best solution to the problem.

**Comment by forgie1** [comment]
closed but not solved :(
